2007-02-01  Mike Kronenberg  <mike.kronenberg@kronenberg.org>

	* Fix for QEMU 0.9.0.
	
2006-06-12  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

	* Try to enforce one exit point per synthetic opcode (gcc4).

--- qemu-0.8.0/target-i386/op.c.i386-FORCE_RET	2005-12-19 23:51:53.000000000 +0100
+++ qemu-0.8.0/target-i386/op.c	2006-02-12 17:51:40.000000000 +0100
@@ -1032,6 +1032,7 @@ void OPPROTO op_aaa(void)
     }
     EAX = (EAX & ~0xffff) | al | (ah << 8);
     CC_SRC = eflags;
+    FORCE_RET();
 }
 
 void OPPROTO op_aas(void)
@@ -1056,6 +1057,7 @@ void OPPROTO op_aas(void)
     }
     EAX = (EAX & ~0xffff) | al | (ah << 8);
     CC_SRC = eflags;
+    FORCE_RET();
 }
 
 /* segment handling */
